steampy/confirmation.py

```diff
@@ -82,14 +91,30 @@ def _create_confirmation_params(self, tag_string: str) -> dict:
                 'm': 'android',
                 'tag': tag_string}
 
-    def _select_trade_offer_confirmation(self, confirmations: List[Confirmation]) -> Confirmation:
+    def _select_trade_offer_confirmation(self, confirmations: List[Confirmation], trade_offer_id: str) -> Confirmation:
         for confirmation in confirmations:
             confirmation_details_page = self._fetch_confirmation_details_page(confirmation)
             confirmation_id = self._get_confirmation_trade_offer_id(confirmation_details_page)
-            if confirmation_id == self._trade_offer_id:
+            if confirmation_id == trade_offer_id:
                 return confirmation
         raise ConfirmationExpected
 
+    def _select_sell_listing_confirmation(self, confirmations: List[Confirmation], asset_id: str) -> Confirmation:
+        for confirmation in confirmations:
+            confirmation_details_page = self._fetch_confirmation_details_page(confirmation)
+            confirmation_id = self._get_confirmation_sell_listing_id(confirmation_details_page)
+            if confirmation_id == asset_id:
+                return confirmation
+        raise ConfirmationExpected
```

Comment:
_select_trade_offer_confirmation and _select_sell_listing_confirmation looks similar, when you unifu asset_id and trade_offer_id as one expected_confirmation id parameter it can be one method